src/content/docs/start/prerequisites.mdx

```diff
@@ -254,6 +273,19 @@ It's important to restart your Terminal to ensure it recognizes the new installa
 
 While npm is the default package manager for Node.js, you can also use others like pnpm or yarn. To enable these, run `corepack enable` in your Terminal. This step is optional and only needed if you prefer using a package manager other than npm.
 
+  </TabItem>
+  <TabItem label="Nix/NixOS">
+  Add the following to your `buildInputs` to install Nodejs:
+  ```nix
+  nodejs # feel free to change the version
+  npm
+  ```
+
+    While npm is the default package manager for Node.js, you can also use others like pnpm or yarn. To install them look for that package on [nixpkgs](https://search.nixos.org/packages) and add the package above.  This step is optional and only needed if you prefer using a package manager other than npm.
+
+  </TabItem>
+</Tabs>
+
```

Comment:
I don't think this is worthy of a dedicated section. It would be much better to just have comments in the dev shell shown above again, as projects will always need a version of Nodejs and a package manager
```suggestion
```